### ISSUE TITLE
nvda does not read list of screens when sharing in Microsoft Teams

### ISSUE THREAD

**Issue by hamidrezaabroshan** [nvaccess/nvda#15341]
### Steps to reproduce:

call someone in ms teams and press ctrl+shift+e to activate share screen.
press tab to move to a section that allows to choose screen to share and Nvda does not read list of screens when pressing arrow keys or tab.
This is very important because it causes for user difficulty when want to collaborate with someone.
This screen was accessible before and now narrator still can read this section.
### Actual behavior:
explained before.
### Expected behavior:
press ctrl+shift+e in a call and press tab to move to share screen section. you can choose the screen you want to share with arrow keys and by pressing enter that option will be activated.

#### NVDA installed/portable/running from source:
installed.
#### NVDA version:
2023.1
#### Windows version:
windows 10
#### Name and version of other software in use when reproducing the issue:
Microsoft teams.
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by thgcode** [comment]
I also have this bug. I think it could be caused by an update on teams itself, as I didn't update NVDA when it started happening. However, an work around that worked for me is: press ctrl+shift+e. The share screen dialog will appear and NVDA will switch to browse mode. Press NVDA+space to switch it back to focus mode. Press tab and it will read normally.

**Comment by seanbudd** [comment]
Have you reported this to Microsoft yet?

**Comment by seanbudd** [comment]
Which version of MS Teams is this?

**Comment by seanbudd** [comment]
Closing as abandoned. Required information has not been supplied.
